### DISCUSSION TITLE
Interactive 3D visualization of embeddings

### DISCUSSION THREAD

**Discussion by namedgraph** [openai/openai-python#856] [General]
Hi,

I know there's this example here: https://github.com/openai/openai-cookbook/blob/main/examples/Visualizing_embeddings_in_3D.ipynb

But I'm looking for the interactive 3D viewer code as shown in [this blog post](https://openai.com/blog/introducing-text-and-code-embeddings). Is it available somewhere?

**Comment by rattrayalex** [comment]
That sounds like a great [feature request to make on the cookbook repo](https://github.com/openai/openai-cookbook/issues/new?assignees=&labels=&projects=&template=feature_request.md&title=%5BFEATURE%5D)!
